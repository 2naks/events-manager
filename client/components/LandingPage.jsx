import React, { Component } from 'react';
import NavBar from './NavBar.jsx';
import Footer from './Footer.jsx';
import Body from './Body.jsx';

/**
 * React  component for landing page
 */
export default class LandingPage extends Component {
  /**
   * React's method to render react component.
   * Renders landing page
   *
   * @return {object}
   */
  render() { //eslint-disable-line class-methods-use-this
    return (
      <div style={{ maxWidth: "100%", overflowX: "hidden" }}>
        <NavBar page="landingpage"/>
        <Body />
        <Footer />
      </div>
    );
  }
}
