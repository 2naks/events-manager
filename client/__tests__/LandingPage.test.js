import React, { Components } from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import LandingPage from '../components/LandingPage.jsx';

Enzyme.configure({ adapter: new Adapter() });

describe('Landing Page Component', () => {
  it('it should render landing page', () => {
    const { wrapper } = shallow(<LandingPage />);
  });
});
