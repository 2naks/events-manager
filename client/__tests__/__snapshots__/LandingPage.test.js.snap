// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page Component it should render landing page 1`] = `undefined`;
